invenio/legacy/bibcheck/__init__.py

```diff
@@ -23,6 +23,13 @@
 This API lets other modules interact with bibcheck.
 """
 
+import warnings
+
+from invenio.utils.deprecation import RemovedInInvenio22Warning
+
+warnings.warn("Legacy BibCheck will be removed in 2.2.",
+              RemovedInInvenio22Warning)
+
```

Comment:
@dset0x, @jalavik: @tiborsimko is right! We should factor out the new checker into an independent module.